### ISSUE TITLE
cannot delete release that are in FAILED state

### ISSUE THREAD

**Issue by sebgoa** [kubernetes/helm#1333]
I have some releases in FAILED state, and it seems that I cannot remove them, even with `--purge`:

```
$ helm version
Client: &version.Version{SemVer:"v2.0.0-alpha.5", GitCommit:"df30c4ab60e90dcdcdb101c6b345f88c49969806", GitTreeState:"dirty"}
Server: &version.Version{SemVer:"v2.0.0-alpha.5", GitCommit:"4de5c4aa578ed9ca173aff472f00d2167ec59a88", GitTreeState:"dirty"}
sebair: examples $ helm list --all
NAME          	REVISION	UPDATED                 	STATUS  	CHART                 
jaundiced-gori	1       	Mon Oct 10 22:41:56 2016	DEPLOYED	owncloud-0.2.0        
jazzed-blackbi	1       	Mon Oct 10 21:03:59 2016	DEPLOYED	redis-standalone-0.0.1
jazzed-shark  	1       	Tue Oct 11 17:30:28 2016	FAILED  	owncloud-0.2.0        
jolly-tortoise	1       	Tue Oct 11 17:28:52 2016	FAILED  	owncloud-0.2.0        
ponderous-mars	1       	Mon Oct 10 21:33:26 2016	DELETED 	consul-0.1.0          
sullen-moose  	1       	Mon Oct 10 21:08:05 2016	DEPLOYED	owncloud-0.2.0        
tinseled-narwh	1       	Mon Oct 10 21:07:45 2016	DEPLOYED	nginx-alpine-0.1.0    
turbulent-goat	1       	Tue Oct 11 17:34:14 2016	DEPLOYED	owncloud-0.2.0        
sebair: examples $ helm delete --purge jazzed-shark
Error: release: not found
sebair: examples $ helm delete --purge jolly-tortoise
Error: release: not found
```

Also related is the inability to remove releases that appear in DEPLOYED state but that I deleted manually out of band from helm.

**Comment by technosophos** [comment]
Blocked until #1339 gets merged

**Comment by fibonacci1729** [comment]
fixed by #1380
